src/script/luna/lunaimgbox.cpp

```diff
@@ -18,7 +18,9 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#ifndef PGE_NO_THREADING
 #include <atomic>
+#endif
```

Comment:
It seems this header is not used here at all, feel free to completely get rid of it.